sql/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -381,17 +382,32 @@ public void testUpdateNestedObjectDeleteWithoutArgs() throws Exception {
     }
 
     @Test
-    public void testUpdateWithNestedObjectArrayIdxAccess() throws Exception {
```

Comment:
Don't remove this test (with the range), maybe move it to SelectStatementAnalyzerTest.
@seut what do you think?